src/dosbox.cpp

```diff
@@ -596,6 +596,13 @@ void DOSBOX_Init(void) {
 	Pint->SetMinMax(0,100);
 	Pint->Set_help("How many milliseconds of data to keep on top of the blocksize.");
 
+// Registered MIDI handlers before running MIDI_Init(), which queries them.
+#if C_FLUIDSYNTH
+	FLUID_AddConfigSection(control); // a MIDI handler
+#endif
+#if C_MT32EMU
+	MT32_AddConfigSection(control); // a MIDI handler
+#endif
```

Comment:
This has bad side-effect of moving `[fluidsynth]` and `[mt-32]` sections *before* `[midi]` section in the .conf file, which makes it a bit problematic…

Reply:
Thanks for catching this; and we certainly don't want the order to change.